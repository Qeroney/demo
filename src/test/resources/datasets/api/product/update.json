{
  "product": [
    {
      "id": "a8c707fa-8a83-4ed0-bebf-1c407d68895f",
      "title": "Огурец",
      "price": 12,
      "category_id":"824c60d2-6c72-4822-9adf-a7e3b53938b8"
    }
  ],
  "category": [
    {
      "id": "824c60d2-6c72-4822-9adf-a7e3b53938b8",
      "title": "Овощи"
    },
    {
      "id": "9b8af09b-4dee-41a6-b608-819f4978958e",
      "title": "Фрукты"
    }
  ]
}