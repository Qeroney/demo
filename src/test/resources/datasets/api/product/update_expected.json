{
  "product": [
    {
      "id": "a8c707fa-8a83-4ed0-bebf-1c407d68895f",
      "title": "Клубника",
      "price": 50,
      "category_id":"9b8af09b-4dee-41a6-b608-819f4978958e"
    }
  ],
  "category": [
    {
      "id": "824c60d2-6c72-4822-9adf-a7e3b53938b8",
      "title": "Овощи"
    },
    {
      "id": "9b8af09b-4dee-41a6-b608-819f4978958e",
      "title": "Фрукты"
    }
  ]
}